openqa-schedule-mm-ping-test

```diff
@@ -53,7 +54,8 @@ job_templates:
       PARALLEL_WITH: ping_server
 EOF
 
-hdd=$(runcli openqa-cli api --host "$openqa_url" jobs version="$version" scope=relevant arch="$arch" flavor="$flavor" test="$test_name" latest=1 | runjq -r '.jobs | map(select(.result == "passed")) | max_by(.settings.BUILD) .settings.HDD_1')
```

Comment:
This is just HDD selection, we cannot use HDD_1 from obsoleted/skipped hosts, right?